### ISSUE TITLE
Source maps

### ISSUE THREAD

**Issue by jwulf** [dcodeIO/MetaScript#9]
Yo, any chance that you could add sourcemaps support to this? 

I'd like to use it with wallaby.js, but wallaby needs a source map to operate. Ref: https://github.com/wallabyjs/public/issues/156

**Comment by dcodeIO** [comment]
I agree that sourcemaps would make this package a lot more usable, but I am currently not planning to invest in this.
